openmls/src/framing/private_message.rs

```diff
@@ -129,8 +129,7 @@ impl PrivateMessage {
         )
     }
 
-    /// Internal function to encrypt content. The extra message header is only used
-    /// for tests. Otherwise, the data from the given `AuthenticatedContent` is used.
+    /// Internal function to encrypt content.
```

Comment:
Is there a particular reason you're removing the remark about the extra header here?

Reply:
It seemed outdated and no longer necessary.

Reply:
It does (still) describe what's going on in the code and I would say it's useful for anyone who is looking to understand what this function does.